### ISSUE TITLE
As a honda client, I would like to be able to choose a font for the matchup page

### ISSUE THREAD

**Issue by jordanskole** [autorivalry/ar-render#22]
As a honda client, I would like the font to match my existing website, and to be able to choose the font, and the font size for the matchup page.

**Comment by jordanskole** [comment]
Hey @sminke , how  important is this to client? Is it a must-have, or could they pay and then we roll it out?

**Comment by sminke** [comment]
@jordanskole I don't think this is that important right now.  No need to apply resources to it until agencies start to bring it up.  Thanks!

**Comment by jordanskole** [comment]
cool, I'll leave it in backlog until it becomes more pressing.
